ch.sbb.scion.rcp.microfrontend/src/ch/sbb/scion/rcp/microfrontend/browser/DisposableJsFunction.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Project: RCS - Rail Control System
+ *
+ * © Copyright by SBB AG, Alle Rechte vorbehalten
+ */
+package ch.sbb.scion.rcp.microfrontend.browser;
+
+/**
+ * A JavaScript function which should be disposed of to free up resources when no longer needed.
+ */
+public interface DisposableJsFunction {
+
+  /**
+   * Disposes the function and releases all allocated resources. This method should be called when the function is no longer needed to
```

Comment:
Please keep the first paragraph to be a single sentence. Please put all the rest to the subsequent paragraph(s).

The first sentence should answer to the question "What is it?" or "What it does?". If readers want to know more, they will go further.

Right now, readers will have to parse the next sentence in any case. However, these details may not be needed.